charts/pgadmin4/values.yaml

```diff
@@ -2,9 +2,17 @@
 
 replicaCount: 1
 
+## Define values for tests/
```

Comment:
Since this only related to [chart tests](https://helm.sh/docs/topics/chart_tests/), I think it would be logical to move this section to the bottom of the file or at least after `image:` with some comment about what does it affect.

Reply:
updated. Please suggest the exact text if the comment is not clear enough.